plugins/okta/help.md

```diff
@@ -1612,7 +1612,6 @@ by Okta themselves, or constructed by the plugin based on the information it has
 
 # Version History
 
-* 4.2.4 - Fix schemas that contain passwords
 * 4.2.3 - Monitor Logs task: Added exception logging and use latest plugin SDK.
```

Comment:
should we mention the schema change in the change log as well?

Reply:
Yep good call @joneill-r7 I'l update that now 👍